src/main/java/org/spongepowered/api/entity/player/Player.java

```diff
@@ -144,4 +144,17 @@
      * @return The connection
      */
     PlayerConnection getConnection();
+
+    /**
+     * Kicks the player.
```

Comment:
`Kicked by an operator.` is the default.

>Oh mail delivery.

Probably to much change notifications in the Attribute PR